pkg/api/api.go

```diff
@@ -15,7 +15,7 @@ const (
 	OpenShiftConsoleConfigMapName           = "console-config"
 	OpenShiftConsolePublicConfigMapName     = "console-public"
 	ServiceCAConfigMapName                  = "service-ca"
-	DefaultIngressCertConfigMapName         = "default-ingress-cert"
+	DefaultIngressCertConfigMapName         = "oauth-serving-cert"
```

Comment:
We'd want to rename this const to make it clear it's not the default ingress cert now.